variables.tf

```
@@ -45,6 +45,12 @@ variable "subnets" {
   description = "The list of subnets being created"
 }
 
+variable "subnet_log_config" {
+  type        = "map"
```

Because this variable has an expected structure, it should be defined as an object rather than a map.

```suggestion
  type        = object({ aggregation_interval = string, flow_sampling = number, metadata = string })
```